### DISCUSSION TITLE
Browser integration/connection lost. Configuration unavailable in KeepassXC settings.

### DISCUSSION THREAD

**Discussion by 88azerty** [keepassxreboot/keepassxc#10686] [Q&A] [answered]
Hello Everyone!

Today when trying to autofill a password I noticed that the Browser Integration failed to detect an open KeepassXC Instance. The option worked without issues for a long time.

To try to solve the issue, I removed the existing connection inside of the KeepassXC-Browser extension settings. When trying to recreate it, the standalone running KeepassXC was not detected.

Then I tried to check the settings with my database opened and the only two entries in the "Application Settings" (Tools > Settings) area in KeepassXC are:

- General
- Security

No other section is available. I tried searching the web for a fix and found no helpful information.

Debug information follows:

```
KeePassXC - Version 2.7.7
Revision: 68e2dd8

Qt 5.15.10
Debugging mode is disabled.

Operating system: Debian GNU/Linux trixie/sid
CPU architecture: x86_64
Kernel: linux 6.7.12-amd64

Enabled extensions:
- Auto-Type

Cryptographic libraries:
- Botan 2.19.4
```

Any helpful tip would be very welcome. Thank you for your time!

~hcg

**Comment by droidmonkey** [comment, marked as answer]
Please check out our troubleshooting guide: https://github.com/keepassxreboot/keepassxc-browser/wiki/Troubleshooting-guide

**Comment by 88azerty** [comment]
Thank you droidmonkey! I reviewed my distro's repository (salsa.debian.org) for KeepassXC and the issue is located in a packaging change. 

For any other person having the same issue: Debian moved the KeepassXC extension required for browser integration to the keepassxc-full package.

**Reply by droidmonkey** [reply]
Oh yah, and that.... terrible decision. I recommend complaining to Debian about ripping functionality out of the app under the same package name. They should have made a keepassxc-min instead.

**Reply by 88azerty** [reply]
Done! There was already a different user with similar issues, I added your recommendation to the thread (https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1069743)
